docs/interactions/structured-messages/select.md

```diff
@@ -284,14 +285,11 @@ Primary parameters are used by default, however, some parameters are unique or o
 | **`structured_content.button`** | String | **Optional**. The button text field.<br>Limited to 20 characters. "See options" by default. |
 | **`structured_content.title`** | String | **Optional**. The title text field.<br>Limited to 60 characters.  |
 | **`structured_content.footer`** | String | **Optional**. The footer text field.<br>Limited to 60 characters.  |
-|
 | **Section Settings** | | |
 | **`structured_content.sections`** | Array | **Optional**. Limited to 10 elements. |
 | **`structured_content.sections.title`** | String | **Optional if there's only a single section**. The title of the section.<br>Limited to 24 characters. |
 | **`structured_content.sections.identifier`** | String | Identifier of the section that will be used to organize items in the section.<br>Limited to 200 characters. |
-|
 | **Item Settings** | | |
 | **`structured_content.items.title`** | String | The item title field.<br>Truncated to 24 characters. |
-| **`structured_content.items.payload`** | String | **Optional**. The item payload field.<br>Limited to 200 characters.<br>Automatically gets populated as a random hex if blank. |
```

Comment:
We still need this one because the limit is different from the generic payload.

Reply:
And you can probably say it's autopopulated too.

Reply:
ah, I removed it accidentally, fixed